apps/omar-ui-app/grails-app/conf/application.yml

```diff
@@ -208,7 +208,7 @@ banner:
   text: BETA
   textColor: white
 
-baseUrl: https://omar-dev.ossim.io
+baseUrl: https://omar-3pa-dev.ossim.io
```

Comment:
Probably should put this back to the way it was, and get rid of the federation configuration.

Reply:
Done.